package.json

```diff
@@ -4,6 +4,7 @@
   "license": "MIT",
   "dependencies": {
     "@cfworker/json-schema": "*",
+    "@exodus/schemasafe": "^1.0.0-alpha.4",
```

Comment:
I general I use `*` for the latest stable version. This should be changed once you reach a stable v1.